src/Http/Http.Results/src/Results.cs

```diff
@@ -978,6 +979,49 @@ public static IResult AcceptedAtRoute<TValue>(string? routeName, RouteValueDicti
 #pragma warning restore RS0026 // Do not add multiple public overloads with optional parameters
         => value is null ? TypedResults.AcceptedAtRoute(routeName, routeValues) : TypedResults.AcceptedAtRoute(value, routeName, routeValues);
 
+    /// <summary>
+    /// Produces a <see cref="ServerSentEventsResult{TValue}"/> response.
+    /// </summary>
+    /// <param name="values">The values to be included in the HTTP response body.</param>
+    /// <param name="eventType">The event type to be included in the HTTP response body.</param>
+    /// <returns>The created <see cref="ServerSentEventsResult{TValue}"/> for the response.</returns>
+    /// <remarks>
+    /// Strings serialized by this result type are serialized as raw strings without any additional formatting.
+    /// </remarks>
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
+    public static IResult ServerSentEvents(IAsyncEnumerable<string> values, string? eventType = null)
```

Comment:
Event types are values scoped to individual events, and not the entire stream. Given that the `SseItem<T>` overload hardcodes to serialization, how can this API produce raw SSE events that includes other per-event fields such as event types?

I would guess that this was discussed during API review, but have you considered factoring into a separate method group (e.g. `ServerSentEventsRaw`) that also accepts `IAsyncEnumerable<SseItem<string>>`?

Reply:
Nevermind, it seems the comment in line 1004 is addressing my concern.